pkgs/by-name/fl/flexcpp/package.nix

```diff
@@ -0,0 +1,99 @@
+{
+  lib,
+  bisoncpp,
```

Comment:
Line 61.
This is a more or less typical design pattern: we tests reverse dependencies so that we can guarrantee that e.g. updating flexcpp does not break existing bisoncpp.

You can see a similar thing at live555.